core/components/handler/pktStorage.go

```diff
@@ -145,6 +147,7 @@ func (e pktEntry) MarshalBinary() ([]byte, error) {
 	rw := readwriter.New(nil)
 	rw.Write(e.AppEUI)
 	rw.Write(e.DevEUI)
+	rw.Write(uint16(e.FPort))
```

Comment:
see http://forum.thethingsnetwork.org/t/downlink-problem/2433 - Seems to be an issue with downlink at the moment.